main.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
+const navBtn = document.querySelector('.nav-btn');
```

Comment:
- [ ] Please don't disable linter rule checks using inline comments like the above the rules are carefully crafted according to industry standards so this will make your code more professional and up to standards. ✔